mdbenchmark/mdengines/utils.py

```diff
@@ -101,20 +101,29 @@ def analyze_benchmark(engine, benchmark):
     threads = np.nan
     hyperthreading = np.nan
     module = None
+    multidir = np.nan
 
     # search all output files
     output_files = glob(
         os.path.join(benchmark.relpath, PARSE_ENGINE[engine.NAME]["analyze"])
     )
     if output_files:
-        with open(output_files[0]) as fh:
-            performance = parse_ns_day(engine, fh)
-            fh.seek(0)
-            ncores = parse_ncores(engine, fh)
+        for f in output_files:
+            performance = []
+            ncores = []
```

Comment:
You need to put the arrays outside the `for`-loop as they will be redefined with every file, no?
```suggestion
        performance = []
        ncores = []
        for f in output_files:
```

Reply:
yes of course! Sorry for that!